mobile/geth.go

```diff
@@ -86,6 +86,9 @@ type NodeConfig struct {
 	// WhisperEnabled specifies whether the node should run the Whisper protocol.
 	WhisperEnabled bool
 
+	// PeerDiscovery indicates whether the node should participate in p2p discovery
+	PeerDiscovery bool
```

Comment:
How does this get set?

For consistency sake, what are your thoughts of keeping `nodiscovery`?  I think at this point, we all know what flag means.

Reply:
Gets set here in `react-native-geth`: https://github.com/celo-org/react-native-geth/pull/17/files#diff-67cba49e3e2aa654dc9f8955ad86296bR95

Personally I'm fine with going back to `nodiscovery`